test/resolution-independence.js

```diff
@@ -0,0 +1,330 @@
+require('chai').should();
+
+var ResolutionIndependence = require('../lib/resolution-independence'),
+	Less = require('../node_modules/less/lib/less');
+
+describe('resolution-independence options', function () {
+	var RezInd, opts, riPlugin, node;
+
+	before(function () {
+		// TODO: we initialize the plugin with a dummy LESS object for now, so that we can test
+		// specific methods of the plugin (this is done using a mock LESS node) - ideally we will
+		// test with the actual nodes generated by LESS, in case the format changes, but this will
+		// require the likely creation of a plugin to hook into the returned nodes from LESS, in
```

Comment:
yo dawg, i heard you like plugins so i built a plugin so you could test your plugin.